src/serviceWorker.js

```diff
@@ -0,0 +1,136 @@
+    
```

Comment:
what is this?

Reply:
idk i never went in to touch this myself tbh

Reply:
Can you remove this and add the serviceWorker.ts that used to be here?